src/features/bridge/context/Bridge.Context.tsx

```diff
@@ -65,15 +67,22 @@ export const BridgeContext = () => {
   const bridgeErrorHandler = useCallback(
     (_error: unknown) => {
       // @ts-ignore
-      const errorCode = _error.code;
+      const errorCode = _error?.code;
+      // @ts-ignore
+      const errorMethods = _error?.method;
       // @ts-ignore
-      const errorMethods = _error.method;
+      const errorMessage = _error?.message || JSON.stringify(_error);
```

Comment:
you can make error message typed instead of using ts-ignore:

ex.

`const errorMessage = (_error as {message?: string}).message`